### ISSUE TITLE
Ignore empty statements in nonblock-statement-body-position

### ISSUE THREAD

**Issue by fasttime** [eslint/eslint#8205]
**Tell us about your environment**

* **ESLint Version:** 3.17.0
* **Node Version:** 7.7.1
* **npm Version:** 4.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default

I would love to have the new rule `nonblock-statement-body-position` ignore empty statements. Such that this is an error:

```js
/* eslint nonblock-statement-body-position: ["error", "below"] */
while (doSomething()) doSomethingElse();
```

But this is not:

```js
/* eslint nonblock-statement-body-position: ["error", "below"] */
while (doSomething());
```

Maybe adding a new option to the rule.

**Comment by not-an-aardvark** [comment]
Seems reasonable to me. Actually, I don't know of anyone who actually uses this style:

```js
while (doSomething())
  ;
```

So the rule could probably just ignore all empty statements.

**Comment by not-an-aardvark** [comment]
Thanks for your interest in improving ESLint. Unfortunately, it looks like this issue didn't get enough support from the team and so I'm closing it. While we wish we'd be able to accommodate everyone's requests, we do need to prioritize. We've found that issues failing to reach consensus after a long time tend to never do it, and as such, we close those issues. This doesn't mean the idea isn't interesting, just that it's not something the team can commit to.
